base/src/com/google/idea/blaze/base/bazel/BuildSystem.java

```diff
@@ -73,7 +73,26 @@ enum SyncStrategy {
   interface BuildInvoker {
 
     enum Capability {
-      IS_LOCAL, SUPPORTS_CLI, SUPPORTS_PARALLELISM, SUPPORTS_API, SUPPORTS_DBIP
+      /**
+       * Capability to build Android Instrumentation Test APK
+       */
+      BUILD_AIT,
+      /**
+       * Capability to invoke blaze/bazel via CLI
+       */
+      SUPPORT_CLI,
+      /**
+       * Capability to run parallel builds
+       */
+      BUILD_PARALLEL_SHARDS,
+      /**
+       * Capability to run blaze/bazel query command via remote invocation
+       */
+      RUN_REMOTE_QUERIES,
+      /**
+       * Capability to debug Android local test
+       */
+      DEBUG_LOCAL_TEST
```

Comment:
If we need a Javadoc to explain the _name_, then what we actually need is a better name.
```suggestion
      /**
       * Capability to build Android Instrumentation Test APK
       */
      BUILD_AIT,
      /**
       * Capability to invoke blaze/bazel via CLI
       */
      INVOKE_BLAZE_CLI,
      /**
       * Capability to run parallel builds
       */
      BUILD_PARALLEL_SHARDS,
      /**
       * Capability to run blaze/bazel query command via remote invocation
       */
      RUN_REMOTE_QUERIES,
      /**
       * Capability to debug Android local test
       */
      DEBUG_ANDROID_LOCAL_TEST
```